parser/parser.py

```diff
@@ -145,6 +151,66 @@ def p_statement_if(p):
     'statement : IF expression THEN statement ELSE THEN statement END_OF_IF_THEN_ELSE END_OF_STATEMENT'
     p[0] = ('if-statement', p[2], p[4], p[7])
 
+def p_variable_parameters(p):
+    'parameters : NAME LIST_ITEM_SEPARATOR parameters'
+    p[0] = ('parameters', p[1], p[3])
+
+def p_variable_parameters_base(p):
```

Comment:
Should this have the empty option. Then we can have '1 å' as params

Reply:
it doesnt need, i will fix